src/m365/teams/commands/chat/chat-member-add.spec.ts

```diff
@@ -106,7 +106,7 @@ describe(commands.CHAT_MEMBER_ADD, () => {
     assert.deepStrictEqual(postStub.lastCall.args[0].data, requestBody);
   });
 
-  it('adds a member by specifying its userId and share all chat history', async () => {
+  it('adds a member by specifying its userId and share all chat history using includeAllHistory parameter', async () => {
```

Comment:
The same here, not to duplicate tests and just have them with the new option and then add a test to check if deprecation warning message is present